### DISCUSSION TITLE
"error": "error finding user: sql: Scan error on column index 3, name \"confirmation_token\": converting NULL to string is unsupported"

### DISCUSSION THREAD

**Discussion by nicolaygerold** [supabase/supabase#17106] [Questions]
I want to use signInWithOtp. I use a regular signup flow, but even for registered users, who have already confirmed emails, I get the above error, when they use the OTP (magic link). 

Can anyone help out. 

It's a status 500 error.

**Comment by jacobboon** [comment]
Did you ever manage to sort this out? I'm running into exactly this error!

**Comment by macMikey** [comment]
i got it.
there are four columns that need to be set to '' (empty), when you create a record. supabase cannot handle a null in those columns:
confirmation_token, email_change, email_change_token_new, recovery_token

**Comment by saalihou** [comment]
In my case it was because I created users using the seed.sql file and did not set those columns.Setting them to empty string as suggested worked.
